guides/source/active_model_basics.md

```diff
@@ -506,35 +1262,45 @@ Finished in 0.024899s, 240.9735 runs/s, 1204.8677 assertions/s.
 6 runs, 30 assertions, 0 failures, 0 errors, 0 skips
 ```
 
-An object is not required to implement all APIs in order to work with
-Action Pack. This module only intends to guide in case you want all
-features out of the box.
-
 ### SecurePassword
 
-`ActiveModel::SecurePassword` provides a way to securely store any
-password in an encrypted form. When you include this module, a
-`has_secure_password` class method is provided which defines
-a `password` accessor with certain validations on it by default.
+`ActiveModel::SecurePassword` provides a way to securely store any password in
+an encrypted form. When you include this module, a `has_secure_password` class
+method is provided which defines a `password` accessor with certain validations
+on it by default.
 
 #### Requirements
 
-`ActiveModel::SecurePassword` depends on [`bcrypt`](https://github.com/codahale/bcrypt-ruby 'BCrypt'),
-so include this gem in your `Gemfile` to use `ActiveModel::SecurePassword` correctly.
-In order to make this work, the model must have an accessor named `XXX_digest`.
-Where `XXX` is the attribute name of your desired password.
-The following validations are added automatically:
+`ActiveModel::SecurePassword` depends on
+[`bcrypt`](https://github.com/codahale/bcrypt-ruby 'BCrypt'), so include this
+gem in your `Gemfile` to use `ActiveModel::SecurePassword`.
+
+```ruby
+gem "bcrypt"
+```
 
```

Comment:
Maybe we need another subtitle here? it feels like everything is under "requirements":

![Screenshot 2024-02-19 at 13 23 28](https://github.com/Ridhwana/rails/assets/26328/84a4e6cd-238b-4889-9878-7ad819271980)

Or maybe we just remove the "requirements" subtitle.